README.md

```diff
@@ -5,6 +5,45 @@ The Jenkins GitHub Issues plugin allows you to create a GitHub issues whenever y
 
 See the wiki at https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Issues+Plugin for documentation and installation instructions. Report bugs here: http://dl.vc/jenkins-github-issues-bug (you will need to first create a Jenkins account at https://accounts.jenkins.io/)
 
+When using the great `job-dsl` plugin, you can configure a publisher step as follows:
```

Comment:
I might move this to the wiki instead: https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Issues+Plugin